packages/dcos-integration-test/extra/test_dcos_diagnostics.py

```diff
@@ -506,10 +506,11 @@ def _download_bundle_from_master(dcos_api_session, master_index):
     bundles = _get_bundle_list(dcos_api_session)
     assert bundles
 
-    expected_common_files = ['dmesg-0.output.gz', 'opt/mesosphere/active.buildinfo.full.json.gz',
+    expected_common_files = ['dmesg_-T-0.output.gz', 'opt/mesosphere/active.buildinfo.full.json.gz',
                              'opt/mesosphere/etc/dcos-version.json.gz', 'opt/mesosphere/etc/expanded.config.json.gz',
                              'opt/mesosphere/etc/user.config.yaml.gz', 'dcos-diagnostics-health.json',
-                             'var/lib/dcos/cluster-id.gz']
+                             'var/lib/dcos/cluster-id.gz', 'timedatectl-5.output.gz',
+                             'binsh_-c_cat etc*-release-6.output.gz', 'systemctl_list-units_dcos*-7.output.gz']
 
     # these files are expected to be in archive for a master host
     expected_master_files = ['dcos-mesos-master.service.gz', 'var/lib/dcos/exhibitor/zookeeper/snapshot/myid.gz',
```

Comment:
Please add check for pods endpoint data

Reply:
Not sure why, I have not seen the integration test check for files generated by "HTTPEndpoints" section but only for "LocalFiles" and "LocalCommands" sections